services/account/controllers/AccountController.ts

```diff
@@ -10,7 +11,30 @@ export class AccountController {
     this.repository = new AccountRepository();
   }
 
-  async getAccount(accountNumber) {
-    return await this.repository.find(accountNumber);
+  async getAccount(filter) {
```

Comment:
According to the swagger spec it is an `object`, but for some reason it comes here as a `string`. How can we utilize the `Filter` interface here?